test/addons-napi/test_typedarray/test_typedarray.c

```diff
@@ -95,10 +95,58 @@ napi_value External(napi_env env, napi_callback_info info) {
   return output_array;
 }
 
+napi_value CreateTypedArray(napi_env env, napi_callback_info info) {
+  size_t argc = 2;
+  napi_value args[2];
+  NAPI_CALL(env, napi_get_cb_info(env, info, &argc, args, NULL, NULL));
+
+  NAPI_ASSERT(env, argc == 2, "Wrong number of arguments");
+
+  napi_valuetype valuetype0;
+  NAPI_CALL(env, napi_typeof(env, args[0], &valuetype0));
+
+  NAPI_ASSERT(env, valuetype0 == napi_object,
+    "Wrong type of argments. Expects a typed array as first argument.");
+
+  napi_value input_array = args[0];
```

Comment:
nit: Can you declare `napi_value input_array = args[0];` and `napi_value input_buffer = args[1];` around line 104 for better readability and reuse it on line 106 and 119 respectively?